src/DateTimeFormatter.php

```diff
@@ -53,7 +53,7 @@ public function formatDiff(
      *
      * @source https://github.com/symfony/symfony/blob/ad72245261792c6b5d2db821fcbd141b11095215/src/Symfony/Component/Console/Helper/Helper.php#L97
      */
-    public function formatDuration(float $seconds, string $locale = null): string
+    public function formatDuration(float $seconds, int $precision = 2, string $locale = null): string
```

Comment:
What about

```suggestion
    public function formatDuration(float $seconds, string $locale = null, ?int $precision = null): string
```

Adding the `$precision` before `$locale` is a BC break. Default to `null` to keep current behavior by default.